src/extras/gizmo/rotate-gizmo.js

```diff
@@ -436,9 +431,19 @@ class RotateGizmo extends TransformGizmo {
             tmpQ1.setFromAxisAngle(tmpV1, angleDelta);
 
             if (!isFacing && this._coordSpace === GIZMOSPACE_LOCAL) {
+                const rot = this._nodeLocalRotations.get(node);
+                if (!rot) {
+                    continue;
+                }
+    
```

Comment:
Please remove extra lines to pass linting

Reply:
Removed